pom.xml

```diff
@@ -137,9 +144,39 @@
         <version>3.0.0</version>
       </dependency>
       <dependency>
-          <groupId>org.jenkins-ci.ui</groupId>
-          <artifactId>jquery-detached</artifactId>
-          <version>1.2.1</version>
+        <groupId>org.jenkins-ci.ui</groupId>
+        <artifactId>jquery-detached</artifactId>
+        <version>1.2.1</version>
+      </dependency>
+      <dependency>
+        <groupId>org.yaml</groupId>
+        <artifactId>snakeyaml</artifactId>
+        <version>1.27</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-yaml</artifactId>
+        <version>2.12.1</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
+        <version>3.9</version>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okhttp3</groupId>
+        <artifactId>okhttp</artifactId>
+        <version>3.14.9</version>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okhttp3</groupId>
+        <artifactId>logging-interceptor</artifactId>
+        <version>3.14.9</version>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okio</groupId>
+        <artifactId>okio</artifactId>
+        <version>1.17.5</version>

```

Comment:
Replace this with the okhttp-api plugin.  This dependency will also be pulled in by github-branch-source.

Reply:
I've tried that, but i still get lots of enforcer errors.  It looks like I need both the plugin and the dependencies defined

Reply:
@garethjevans 
That probably means that there are some things that loading the okhttp3 library and you're probably still packaging it. Could you double check your hpi-plugin output?